examples/example_plugin/example_plugin/jobs.py

```diff
@@ -44,4 +44,15 @@ def run(self, data, commit):
         return f"Ran for {interval} seconds"
 
 
-jobs = (ExampleJob, ExampleHiddenJob, ExampleLoggingJob)
+class ExampleJobHookReceiver(JobHookReceiver):
+    class Meta:
+        name = "Example job hook receiver"
+        description = "This job is triggered by object change events"
+
+    def run(self, data, commit):
+        change_action = data["object_change"].action
+        changed_object = data["object_change"].changed_object
+        self.log_success(message=f"{change_action} detected for object {changed_object.name}")
```

Comment:
Perhaps, since not all models have a `name`, just use the string representation?

```suggestion
        self.log_success(message=f"{change_action} detected for object {changed_object}")
```